### DISCUSSION TITLE
How do you model circuits when you are the provider?

### DISCUSSION THREAD

**Discussion by dejantep** [netbox-community/netbox#15113] [Q&A]
Netbox documentation states following "_Sometimes you'll need to model provider networks into which you don't have full visibility; these are typically represented on topology diagrams with cloud icons._".

This is straight forward logical when i'm the customer in case of ip transit or MPLS. But what about if i'm the _provider_ and i would like to model _provider network_ or more correctly _customer network_?
I know this sounds trivial and i'm free to apply this feature anyway i want but calling it _provider network_ is confusing. I'm the provider of the circuit.

How do you go about it?

**Comment by candlerb** [comment]
Unless you're transiting one of your customer networks, I don't think this is necessary.

You can mark each circuit in a way to identify the customer - for example using Tenant. You connect the A-end to your own network, and the Z-end needs no termination (unless you're providing the CPE, in which case the Z-end goes to the CPE device)

**Comment by dejantep** [comment]
It's kind of what i was afraid of since i'm not transiting customer network.
The main issue that i have with that feature or lack of it is that documentation feels incomplete. If look at a device interface overview i can see it is connected but is not clear to what. It would be nice to see something similar to _provider network_  

I guess it just the way it is for now. Thanks for the reply.
